.github/workflows/check-dist.yml

```diff
@@ -0,0 +1,43 @@
+# `dist/index.js` is a special file in Actions.
+# When you reference an action with `uses:` in a workflow,
+# `index.js` is the code that will run.
+# For our project, we generate this file through a build process
+# from other source files.
+# We need to make sure the checked-in `index.js` actually matches what we expect it to be.
+name: Check dist/
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+  check-dist:
+    runs-on: ubuntu-latest
```

Comment:
Part of me wants this to be `macos-latest` to maintain consistency with the machines we use, but part of me says this should be fine. Only using Windows would cause issues. Your call 🤷‍♂️

Reply:
Ubuntu is lower COGS and yeah should be the same.  If we see friction from this we can change it.

Reply:
also macOS has a lower concurrency limit